modules/sns-topic/README.md

```diff
@@ -0,0 +1,153 @@
+# Component: `sns-topic`
+
+This component is responsible for provisioning an SNS topic.
+
+## Usage
+
+**Stack Level**: Regional
+
+Here are some example snippets for how to use this component:
+
+`stacks/catalog/sns-topic/defaults.yaml` file (base component for all SNS topics with default settings):
+
+```yaml
+components:
+  terraform:
+    sns-topic-defaults:
```

Comment:
```suggestion
    sns-topic/defaults:
```

Reply:
hey @nitrocode, I was following the latest pattern and using `s3-bucket` as an example, which uses something similar to what I have -- https://github.com/cloudposse/terraform-aws-components/tree/master/modules/s3-bucket

Have a look and let me know what you think.

Reply:
I prefer the forward slash since this creates the nice folder structure in spacelift :)

Reply:
It's also more intuitive IMHO because `sns-topic/defaults` matches the catalog file path

Reply:
Fair enough.. I'll make the change.